net/pfSense-pkg-Quagga_OSPF/files/usr/local/pkg/quagga_ospfd.xml

```diff
@@ -117,8 +117,15 @@
 		<field>
 			<fielddescr>Disable FIB updates (Routing table)</fielddescr>
 			<fieldname>updatefib</fieldname>
-			<description>Disables the updating of the host routing table(turns into stub router).</description>
-			<type>checkbox</type>
+			<description>Disables the updating of the host routing table (turns into stub router).</description>
+			<type>select</type>
+			<default_value>none</default_value>
+			<options>
+				<option><name>None (FIB updates enabled) (default)</name><value>none</value></option>
+				<option><name>Stub Area (stub)</name><value>stub</value></option>
+				<option><name>Totally Stub (no-summary)</name><value>stub no-summary</value></option>
```

Comment:
Shouldn't value be "nosum" here? Based on check done in https://github.com/pfsense/FreeBSD-ports/pull/233/files#diff-238d7552fa57444efc0f66cb1bd2b8d3R191